source/_components/plant.markdown

```diff
@@ -27,9 +32,42 @@ plant:
     min_temperature: 15
 ```
 
-## Using plain MQTT sensor to get the data
+Configuration variables:
+
+- **simulated_plant** (*Required*):
+  - **sensors** (*Required*): 
+    - **moisture** (*Optional*):  
+    - **battery** (*Optional*):
+    - **temperature:** (*Optional*):
+    - **conductivity:** (*Optional*):
+    - **brightness:** (*Optional*):
+  - **min_moisture** (*Optional*):
+  - **max_moisture** (*Optional*):
+  - **min_battery** (*Optional*):
+  - **min_conductivity** (*Optional*):
+  - **min_temperature** (*Optional*):
```

Comment:
Add max_moisture, max_conductivity, min_brightness, max_brightness and max_temperature.
Small detail that I am not sure if it should be included: Temperature is float while the others are positive integers.